react/react1/week2/todo-app/src/components/Counter.js

```diff
@@ -0,0 +1,13 @@
+import { useEffect, useState } from 'react';
+
+export default function Counter() {
+    const [seconds, setSeconds] = useState(0);
+
+    useEffect(() => {
+        setTimeout(() => {
+            setSeconds(seconds + 1);
+        }, 1000);
+    });
```

Comment:
I think this useEffect should have a dependency! Figure that out!

Reply:
I added seconds as dependency. 😄